python-base/ubuntu20.04-pyston2.2/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:ubuntu20.04
```

Comment:
What does this do?

Reply:
Created a new ubuntu image with an updated `OpenSSL` -- now extending the base images from the new ubuntu image.

Reply:
No it's not.